src/Color.h

```diff
@@ -51,8 +51,8 @@ class cColor
 
 	/** Resets the color */
 	void Clear() { m_Color = static_cast<unsigned int>(COLOR_NONE); }
-	// tolua_end
```

Comment:
This makes the `m_Color` member exported, probably not what we want. Keep this and remove the one after the class declaration.

Note: `// tolua_begin` and `// tolua_end` mark blocks that are parsed by ToLua, `// tolua_export` marks a single line that is parsed. ToLua basically first removes everything outside of these, and then tries to parse it as a regular C++ code.